solutions/usaco-760.mdx

```diff
@@ -7,10 +7,11 @@ author: Jesse Choe
 
 [Official Analysis](http://www.usaco.org/current/data/sol_shuffle_bronze_dec17.html)
```

Comment:
```suggestion
[Official Analysis Java and C++](http://www.usaco.org/current/data/sol_shuffle_bronze_dec17.html)
```
I think you should add the language to the editorial.
@SansPapyrus683 If there's another way to format an editorial with two languages lmk. 👍

Reply:
I would prefer `Official Analysis (Java and C++)` just to be more consistent but it's up to you.

Reply:
yea that works